content/references/rippled-api/transaction-formats/transaction-results/tef-codes.md

```diff
@@ -23,6 +23,7 @@ These codes indicate that the transaction failed and was not included in a ledge
 | `tefNO_AUTH_REQUIRED`  | The [TrustSet transaction][] tried to mark a trustline as authorized, but the `lsfRequireAuth` flag is not enabled for the corresponding account, so authorization is not necessary. |
 | `tefNOT_MULTI_SIGNING` | The transaction was [multi-signed](multi-signing.html), but the sending account has no SignerList defined. |
 | `tefPAST_SEQ`           | The sequence number of the transaction is lower than the current sequence number of the account sending the transaction. |
+| `tefTOO_BIG`            | The transaction would affect too many objects in the ledger. For example, this was an [AccountDelete transaction][] but the account to be deleted owns over 1000 deletable objects in the ledger. The transaction could succeed on retry if some of those objects were deleted first. |
```

Comment:
This is a bit confusing to me. Do you mean to say 
"but the account to be deleted owns over 1000 deletable objects in the ledger."
"but the account to be deleted owns over 1000 objects that would be deleted by this action in the ledger."

It's confusing because the fact that this object contains that many objects that would be deleted by this action, actually makes this action **un-deletable**. I'm not sure is this is an actual error message or just a description.